python/cudf/cudf/core/frame.py

```diff
@@ -882,11 +882,11 @@ def scatter_by_map(
             )
 
         if kwargs.get("debug", False) == 1 and map_size is not None:
-            unique_count = map_index.unique_count()
-            if map_size < unique_count:
+            distinct_count = map_index.distinct_count()
+            if map_size < distinct_count:
                 raise ValueError(
                     "ERROR: map_size must be >= %d (got %d)."
```

Comment:
```suggestion
                    f"ERROR: map_size must be >= {distinct_count} (got {map_size})."
```

This could be changed to `f-strings` and Line 889 can be deleted.